.github/workflows/release-oura.yml

```diff
@@ -0,0 +1,99 @@
+# Create release files
+name: Release
+
+on:
+  release:
+    types: [ published ]
+
+env:
+  DOCKER_IMAGE: radarbase/kafka-connect-rest-oura-source
+
+jobs:
+  uploadBackend:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v3
+      - uses: actions/setup-java@v3
+        with:
+          distribution: temurin
+          java-version: 11
+
+      - name: Gradle cache
+        uses: actions/cache@v3
+        with:
+          # Cache gradle directories
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          # An explicit key for restoring and saving the cache
+          key: ${{ runner.os }}-gradle-${{ hashFiles('gradlew', '**/*.gradle', 'gradle.properties', 'gradle/**') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+
+      # Compile code
+      - name: Compile code
+        run: ./gradlew jar
+
+      # Upload it to GitHub
+      - name: Upload to GitHub
+        uses: AButler/upload-release-assets@v2.0
+        with:
+          files: "*/build/libs/*"
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+
+  # Build and push tagged release docker image
+  docker:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      - uses: actions/checkout@v3
+
+      # Setup docker build environment
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      # Add Docker labels and tags
+      - name: Docker meta
+        id: docker_meta
+        uses: docker/metadata-action@v4
+        with:
+          images: ${{ env.DOCKER_IMAGE }}
+          tags: |
+            type=semver,pattern={{version}}
+            type=semver,pattern={{major}}.{{minor}}
+
+      - name: Build docker
+        uses: docker/build-push-action@v3
+        with:
+          file: ./kafka-connect-oura-source/Dockerfile
```

Comment:
would be good to keep the similar file path for fitbit Dockerfile too for consistency